tests/Microsoft.Fx.Portability.Tests/BreakingChangeParserTests.cs

```diff
@@ -187,7 +241,26 @@ private void TestEquality(BreakingChange expected, BreakingChange actual)
 
         private Stream GetBreakingChangeMarkdown(string resourceName)
         {
-            var name = typeof(BreakingChangeParserTests).GetTypeInfo().Assembly.GetManifestResourceNames().Single(n => n.EndsWith(resourceName, StringComparison.Ordinal));
+            var resources = typeof(BreakingChangeParserTests).GetTypeInfo().Assembly.GetManifestResourceNames();
+            string name = null;
+
+            try
+            {
+                name = resources.Single(n => n.EndsWith(resourceName, StringComparison.Ordinal));
+            }
+            catch (InvalidOperationException)
+            {
+                _output.WriteLine("These are the embedded resources:");
+
+                for (int i = 0; i < resources.Length; i++)
+                {
+                    var resource = resources[i];
+                    _output.WriteLine($"\t{i}: {resource}");
+                }
+
+                throw;
+            }
+
             return typeof(BreakingChangeParserTests).GetTypeInfo().Assembly.GetManifestResourceStream(name);
```

Comment:
This can be moved up so that the `name` variable can have its declaration and definition combined

Reply:
Fixed! :)